code/datums/bodypart_overlays/bodypart_overlay.dm

```diff
@@ -69,7 +69,7 @@
 			return -BODY_FRONT_LAYER
 
 ///Check whether we can draw the overlays. You generally don't want lizard snouts to draw over an EVA suit
-/datum/bodypart_overlay/proc/can_draw_on_bodypart(mob/living/carbon/human/human)
+/datum/bodypart_overlay/proc/can_draw_on_bodypart(obj/item/bodypart/bodypart_owner, mob/living/carbon/human/human)
```

Comment:
Up to you, but I don't think you should even pass `human` as an argument anymore. It can be accessed with `bodypart.owner` for things that actually need it.